build.gradle.kts

```diff
@@ -10,3 +10,12 @@ dependencies {
     implementation("org.openrewrite:rewrite-yaml:${rewriteVersion}")
     runtimeOnly("com.fasterxml.jackson.core:jackson-core")
 }
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+}
+
+tasks.named<JavaCompile>("compileJava") {
+    options.release.set(8)
+}
```

Comment:
Why these changes? Should be set through the plugin already right?
```suggestion
```

Reply:
I can try revert this but  at the my IDE was having issues with the syntax.